terraform-test-user/terraform.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    encrypt        = true
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
+    key            = "skeleton-packer/terraform-test-user.tfstate"
+  }
+}
```

Comment:
This file should be renamed `backend.tf` to match how we do it elsewhere.

Reply:
Good catch, resolved in 8ed3842.